ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -634,7 +634,7 @@ EventReportUrl,string,Optional,RegistryEvent,URL,,
 EventResult,string,Mandatory,RegistryEvent,,,
 EventSchemaVersion,string,Mandatory,RegistryEvent,SchemaVersion,,
 EventStartTime,datetime,Mandatory,RegistryEvent,,,
-EventType,string,Mandatory,RegistryEvent,Enumerated,RegistryKeyCreated|RegistryKeyDeleted|RegistryKeyRenamed|RegistryValueDeleted|RegistryValueSet|RegistryValueRead|MetadataModified|ObjectDeleted|MetadataAccessed|RegistrySubkeyEnumerated,
+EventType,string,Mandatory,RegistryEvent,Enumerated,RegistryKeyCreated|RegistryKeyDeleted|RegistryKeyRenamed|RegistryValueDeleted|RegistryValueSet|RegistryValueRead|MetadataModified|ObjectDeleted|MetadataAccessed|RegistrySubkeyEnumerated|Other,
```

Comment:
Remove ObjectDeleted. We should use RegisteryValueDeleted. I don't think for Registry we have a log that can be either key of value delete.

Reply:
Done